src/theme.c

```diff
@@ -627,6 +627,26 @@ entry(struct theme *theme, const char *key, const char *value)
 			theme->window_inactive_button_close_unpressed_image_color);
 	}
 
+	/* window drop-shadows */
+	if (match_glob(key, "window.active.shadow.radius")) {
+		theme->window_active_shadow_radius = atoi(value);
+	}
```

Comment:
Misses setting the default values in `theme_builtin()`, same reason as for `rcxml.c`.